wrapper/src/main/java/software/amazon/jdbc/targetdriverdialect/GenericTargetDriverDialect.java

```diff
@@ -94,4 +95,13 @@ public boolean isDriverRegistered() throws SQLException {
   public void registerDriver() throws SQLException {
     throw new SQLException(Messages.get("TargetDriverDialect.unsupported"));
   }
+
+  @Override
+  public boolean ping(@NonNull Connection connection) {
+    try {
+      return connection.isValid(10);
```

Comment:
NIT: add comment for indicating the unit

Reply:
That's in seconds and it's part of JDBC spec.